bothub/common/models.py

```diff
@@ -284,6 +291,12 @@ class Meta:
         max_length=32,
         help_text=_("Easy way to found and share repositories"),
     )
+    repository_type = models.CharField(
+        _("repository type"),
+        max_length=50,
```

Comment:
Set the max_length to 10 characters maximum, you have no value of 50 characters to set with that amount, that way you save space in the sg